packages/grafana-data/src/geo/layer.ts

```diff
@@ -102,6 +106,12 @@ export interface MapLayerRegistryItem<TConfig = MapLayerOptions> extends Registr
    */
   showOpacity?: boolean;
 
+  /**
+   * Whether this layer type renders its features from a data frame
+   * Required field as this determines key options layout logic
+   */
+  usesDataFrame: boolean;
```

Comment:
```suggestion
  usesData: boolean;
```
frames seems a bit specific?  or maybe "usesQueryResult?"